src/components/ProposalDiv.tsx

```diff
@@ -0,0 +1,65 @@
+import Image from "next/image";
+import Link from "next/link";
+import { Progress } from "./shadCDN/progress";
+import { FaImage } from "react-icons/fa6";
+import { HomePageAvatar } from "./HomePageAvater";
+
+
+
+export default function ProposalDiv() {
+    return (
+        <Link href={'/proposals/polars-marco'}
```

Comment:
this is hard coded?

Reply:
yes for now it's hard coded, the fucntion to get the file name from proposal directory is not working properly. I will make this dynamic after fixing that.